katas/jorge-chavez/02-13/Sudoku Validator/SudokuValidator.cs

```diff
@@ -0,0 +1,53 @@
+public class SudokuValidator
```

Comment:
Please remove this kata from this PR, I won't be able to review it just yet and we need to merge this PR

Reply:
Done